### PR TITLE
Delete unused SchemaDescription from KT codebase

### PR DESCRIPTION
**Pull request by mmandlis**
descriptions are used in planning, and in suggestions, i don't see us needing it in KT any time soon (unfortunately?)

### CODE REVIEW

File: javatests/arcs/android/host/ArcHostHelperTest.kt [hunk 1]
```diff
@@ -30,8 +30,7 @@ import arcs.core.data.HandleConnectionSpec
 import arcs.core.data.ParticleSpec
 import arcs.core.data.PlanPartition
 import arcs.core.data.Schema
-import arcs.core.data.SchemaDescription
-import arcs.core.data.SchemaFields
+=import arcs.core.data.SchemaFields
```
Comment (csilvestrini) [inline javatests/arcs/android/host/ArcHostHelperTest.kt]: Typo at start of line

File: javatests/arcs/android/host/parcelables/ParcelablePlanTest.kt [hunk 2]
```diff
@@ -17,8 +17,7 @@ import arcs.core.common.ArcId
 import arcs.core.data.EntityType
 import arcs.core.data.FieldType
 import arcs.core.data.Schema
-import arcs.core.data.SchemaDescription
-import arcs.core.data.SchemaFields
+simport arcs.core.data.SchemaFields
```
Comment (csilvestrini) [inline javatests/arcs/android/host/parcelables/ParcelablePlanTest.kt]: Typo at start of line

File: javatests/arcs/android/host/ArcHostHelperTest.kt (hunk 1 above)
Comment (mmandlis) [reply javatests/arcs/android/host/ArcHostHelperTest.kt]: omg, how did that happen?! fixed.

File: javatests/arcs/android/host/parcelables/ParcelablePlanTest.kt (hunk 2 above)
Comment (mmandlis) [reply javatests/arcs/android/host/parcelables/ParcelablePlanTest.kt]: this is truly embarrassing and explains all the failures... fixed.
sorry and thanks.

**Comment by jasonwyatt** [comment]
This will probably need a change internally as well, for the demo app.

**Comment by mmandlis** [comment]
> This will probably need a change internally as well, for the demo app.

good catch, thanks! done that in cl/296090562, added you as reviewer, waiting for the tests to finish.

**Pull request merged**